theme/themes/pastanaga/extras/toolbar.less

```diff
@@ -19,6 +19,70 @@
   }
 }
 
+body:not(.has-sidebar):not(.has-sidebar-collapsed) {
```

Comment:
Do we need the `:not(.has-sidebar):not(.has-sidebar-collapsed)`?

Reply:
yes, for cascading specificity